src/App/mermaidData/databaseSwitchboard/ImageClassificationMixin.js

```diff
@@ -0,0 +1,22 @@
+import axios from '../../../library/axiosRetry'
+import { getAuthorizationHeaders } from '../../../library/getAuthorizationHeaders'
+
+const ImageClassificationMixin = (Base) =>
+  class extends Base {
+    getAnnotationsForImage = async function getAnnotationsForImage(projectId, imageId) {
+      if (!imageId || !projectId) {
+        Promise.reject(this._operationMissingParameterError)
```

Comment:
should return the rejected promise

Reply:
could just use `getAnnotationsForImage = async function` without renaming it again since it's already in a method with the same name

Reply:
ah ok didn't pay attention to the other mixins using that format - doesn't see it as necessary